Misc/NEWS.d/next/Core and Builtins/2019-09-06-16-40-12.bpo-27145.njuCXU.rst

```diff
@@ -0,0 +1 @@
+small_ints[x] could be returned in long_add and long_sub. Patch by hongweipeng.
```

Comment:
small_ints is an implementation detail. Please explain what users get: singletons. Something like:

"int + int and int - int operators can now return small integer singletons."